Core/NosSmooth.Game/Events/Map/MapChangedEvent.cs

```diff
@@ -14,5 +14,5 @@ namespace NosSmooth.Game.Events.Map;
 public record MapChangedEvent
 (
     Data.Maps.Map? PreviousMap,
-    Data.Maps.Map CurrentMap
+    Data.Maps.Map? CurrentMap
```

Comment:
As it's not possible that the map will be changed to null, I would prefer if this stayed non-nullable.

Reply:
Changed back to non-nullable